api/transport/propagation.go

```diff
@@ -37,6 +37,12 @@ type CreateOpenTracingSpan struct {
 	ExtraTags     opentracing.Tags
 }
 
+const (
```

Comment:
You can put tracing related consts here: https://github.com/yarpc/yarpc-go/blob/dev/tracing.go